tests/helpers/rabbitMQ.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-console */
```

Comment:
Can we remove the `console.log` and this `tslint:disable`?

Reply:
yes, only I follow the pattern of other third-party dependencies that we have, like as mongo db. Because the functional test takes a long time and sometimes do you don't know what happens behind the scenes.

what do you say?

Reply:
I feel quite strongly about reducing the output of the application. It should be opt-in at the least, and `console.log` isn't. I understand adding it for debugging and experimenting, but it really shouldn't make it to `master`...

Do you think we'll really really need these? Or did you need these while adding the RMQ functionality but now that it's working we could probably live without it, and add it back if we find stuff breaks?